components/selection/selection-mixin.js

```diff
@@ -38,7 +39,8 @@ export class SelectionInfo {
 			none: 'none',
 			some: 'some',
 			all: 'all',
-			allPages: 'all-pages'
+			allPages: 'all-pages',
+			undefined: 'undefined'
```

Comment:
This new `undefined` selection state is being added to support nested list scenarios where a parent list does not contain any selectables, and we need to treat it as undefined/neutral when evaluating the ancestor selection state.

Reply:
Nit: I'm guessing it's not an issue otherwise it would have come up, but it's a little confusing since `undefined` is a reserved word in JS. Another option would be `notDefined` or `notSet`.

Reply:
Yep... totally open to that. I like both of those better. Any preference?

Reply:
Ok, updated to `notSet`. More searchable too.